backend/routes/queries.py

```
@@ -0,0 +1,342 @@
+"""
+Admin dashboard queries API routes.
+
+Provides endpoints for:
+- Query listing and management
+- Individual query details
+- Query insights and analytics
+"""
+
+import logging
+import sqlite3
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)